make/dev.mk

```diff
@@ -6,9 +6,10 @@ include ./make/git.mk
 
 DOCKER_REPO?=quay.io/openshiftio
 IMAGE_NAME?=devconsole-operator
-REGISTRY_URI=quay.io
+REGISTRY_URI?=quay.io
+REGISTRY_ORG?=redhat-developers
 
-DEVCONSOLE_OPERATOR_IMAGE?=quay.io/redhat-developers/devconsole-operator
+DEVCONSOLE_OPERATOR_IMAGE?=$(REGISTRY_URI)/$(REGISTRY_ORG)/$(IMAGE_NAME)
```

Comment:
@corinnekrych are you using those variables somewhere else or do you absolutely need the ability to set the operator image name from the outside? I mean you could always add a tag to this image as well if you need to customize it. One lesson learned from the fabric8-wit project was to not introduce variables for things you rarely need.